newdle_frontend.js

```diff
@@ -1,41 +1,74 @@
+/* Internal values */
 
+// To be changed with emojis in time.
 const TICK_STRING = "tick for now"
 
 const CROSS_STRING = "cross for now"
 
-const API_INFO = {headline: "Boris Johnson says tens of thousands could be sent to Rwanda under relocation plan for asylum seekers - UK politics live - The Guardian", scrambled_headline: "Bsrio onnJsoh asys snte fo onduathss uolcd be nset ot awnaRd rneud erolniocta pnla rof mluyas sesekre - UK iosplcti ievl - eTh andiuarG"}
+let headlineArray = []
 
-const HEADLINE_ARRAY = API_INFO.headline.split(" ")
-const HEADLINE_ARRAY_LOWER = API_INFO.headline.toLowerCase().split(" ")
+let headlineArrayLower = [] 
 
 const RESULT_PTAG = document.getElementById("result_text")
 
+const GUESS_BOX = document.getElementById("guess_box")
+
+const GUESS_BUTTON = document.getElementById("button")
+
+/* Making boxes unavailable */
+
+GUESS_BOX.disabled = true;
+GUESS_BUTTON.disabled = true;
+
+
+/* External values */
+
+/* fetch initiates the process of getting the information from the API
+.then(api_response) returns the promise of data from the URL. (It takes time to get the data)
+.then(json_response) returns the data in .json format (it takes time to get the data)
+Both retieving data and converting data takes time. That's why .then exists...
+it executes when the data has actually been procured/converted */ 
+fetch('http://localhost:5000/Game_info')
+.then(api_response => {return api_response.json()})
+.then(json_response => {
+    getHeadlines(json_response)
+});
+
+function getHeadlines(headlines_json){
+    document.getElementById("presented_headline").innerText = headlines_json.scrambled_headline
+    headlineArray = headlines_json.headline.split(" ")
+    headlineArrayLower = headlines_json.headline.toLowerCase().split(" ")
+    GUESS_BOX.disabled = false;
+    GUESS_BUTTON.disabled = false;
+}
 
 function onSubmit(){
     checkGuess()
     winningConditions()
 }
 
-
 function checkGuess() {
-    const PLAYER_GUESS = document.getElementById("guess_box").value.toLowerCase()
+    const PLAYER_GUESS = GUESS_BOX.value.toLowerCase()
 
-
-    if (HEADLINE_ARRAY_LOWER.includes(PLAYER_GUESS) == true){
+    if (headlineArrayLower.includes(PLAYER_GUESS) == true && GUESS_BUTTON.hasAttribute('disabled') == false){
```

Comment:
I think this part of the conditions is unnecessary: "&& GUESS_BUTTON.hasAttribute('disabled') == false"

If the button is disabled they will never be able to press it and call this method